hail/python/hail/docs/_static/annotationdb/annotationdb.js

```diff
@@ -0,0 +1,79 @@
+
+
+//CheckAll functionality for UI
+$("#checkAll").click(function(){
+    $('input:checkbox').not(this).prop('checked', this.checked);
+});
+
+//Get and append JSON information to Table
+$.getJSON("https://api.myjson.com/bins/1fn7f7",
+                function (data) {
+                    var tr = data.report
+                    for (var i = 0; i < data.length; i++) {
+                        tr = $('<tr/>');
+                        tr.append("<td><input type='checkbox' class='checkboxadd' value='"+data[i].name+"' onClick='updateTextArea()'/>&nbsp;</td>")
+                        tr.append("<td>" + data[i].name + "</td>");
+                        tr.append("<td>" + data[i].version + "</td>");
+                        tr.append("<td>" + data[i].n_rows + "</td>");
+                        tr.append("<td>" + data[i].n_cols + "</td>");
+                        $('.table1').append(tr);
+                    }
+                });
+
+//Allows filtering of table
+function searchTable() {
+    var input, filter, found, table, tr, td, i, j;
+    input = document.getElementById("searchInput");
+    filter = input.value.toUpperCase();
+    table = document.getElementById("table1");
+    tr = table.getElementsByTagName("tr");
+    for (i = 0; i < tr.length; i++) {
+        td = tr[i].getElementsByTagName("td");
+        for (j = 0; j < td.length; j++) {
+            if (td[j].innerHTML.toUpperCase().indexOf(filter) > -1) {
+                found = true;
+            }
+        }
+        if (found) {
+            tr[i].style.display = "";
+            found = false;
+        } else if (!tr[i].id.match('^tableHeader'))  {
+            tr[i].style.display = "none";
+        }
+    }
+}
+
+//Allows copying of information within textArea
```

Comment:
This function's name and definition is self-documenting, there's no need for this comment.